src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Generated/AvTraceMessages.cs

```diff
@@ -82,7 +82,7 @@ static public void Trace( TraceEventType type, AvTraceDetails traceDetails, para
         /// <summary> These help delay allocation of object array </summary>
         static public void Trace( TraceEventType type, AvTraceDetails traceDetails )
         {
-            _avTrace.Trace( type, traceDetails.Id, traceDetails.Message, traceDetails.Labels, new object[0] );
+            _avTrace.Trace( type, traceDetails.Id, traceDetails.Message, traceDetails.Labels, Array.Empty<object>() );
```

Comment:
This is a generated file (see comment at the top of the file).  You need to make these changes in the template as well, or they'll get overwritten the next time the template changes. 

The right way to do this change is:
a. Change the template file AvTraceMessages.tt
b. Re-run the generator to propagate the changes to the 3 AvTraceMessages.cs files (for Base, Core, and FW).
c. Check in all 4 changed files.
I think step (b) happens automatically at the start of a full build, but you should verify that.

Reply:
@SamBent Thank you and I fix it in https://github.com/dotnet/wpf/pull/2843